.travis.yml

```diff
@@ -2,6 +2,7 @@ language: ruby
 rvm:
   - 2.3.1
   - 2.4.2
+  - 2.5.1
```

Comment:
Should 2.6.1 go in this list?

Reply:
Yea might as well. I gotta fix the rails 6 thing to make sure we exclude < 2.5 in ci